### PR TITLE
eslint-plugin-wdio: add cjs build

### LINKED ISSUE #9312: [🐛 Bug]: `eslint-plugin-wdio` not CJS compatible
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md#reporting-new-issues).

### WebdriverIO Version

latest

### Node.js Version

latest

### Mode

Standalone Mode

### Which capabilities are you using?

```typescript
n/a
```

### What happened?

Using `eslint-plugin-wdio` is not possible in v8 as it is a ESM package while EsLint currently doesn't support ESM. We have to either have it compiled as CJS or provide a CJS alternative at least.

### What is your expected behavior?

Get it back working again.

### How to reproduce the bug.

n/a

### Relevant log output

```typescript
Error [ERR_REQUIRE_ESM]: require() of ES Module /path/to/project/node_modules/eslint-plugin-wdio/build/index.js from /path/to/project/node_modules/@eslint/eslintrc/dist/eslintrc.cjs not supported.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### PR DESCRIPTION
**Pull request by SCG82**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)
Resolves #9312

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: scripts/build.js [hunk 1]
```diff
@@ -97,8 +104,8 @@ if (!HAS_WATCH_FLAG) {
     console.log('Remove `export {}` from CJS files')
     for (const pkg of ['webdriver', 'devtools', 'webdriverio']) {
         const filePath = path.join(__dirname, '..', 'packages', pkg, 'build', 'cjs', 'index.js')
-        const fileContent = await fs.readFileSync(filePath, 'utf8')
-        await fs.writeFileSync(filePath, fileContent.toString().replace('export {};', ''), 'utf8')
+        const fileContent = fs.readFileSync(filePath, 'utf8')
+        fs.writeFileSync(filePath, fileContent.toString().replace('export {};', ''), 'utf8')
```
Comment (christian-bromann) [inline scripts/build.js:108]: Let's keep this and import `node:fs/promises` instead.

```suggestion
        const fileContent = await fs.readFile(filePath, 'utf8')
        await fs.writeFile(filePath, fileContent.toString().replace('export {};', ''), 'utf8')
```

**Review by christian-bromann — COMMENTED** [review]
Thanks for picking this up, one comment.

**Comment by SCG82** [comment]
> Thanks for picking this up, one comment.

done

**Review by christian-bromann — APPROVED** [review]
Great fix, thanks 👍

**Pull request merged**